java/src/jmri/jmrit/symbolicprog/LongAddrVariableValue.java

```diff
@@ -379,30 +356,23 @@ public void propertyChange(@Nonnull java.beans.PropertyChangeEvent e) {
             CvValue cv0 = _cvMap.get(getCvNum());
             CvValue cv1 = highCV;
             int newVal = (cv0.getValue() & 0x3f) * 256 + cv1.getValue();
-            setValue(newVal);  // check for duplicate done inside setVal
+            setValue(newVal);  // check for duplicate done inside setValue
             // state change due to CV state change, so propagate that
             setState(cv0.getState());
             // see if this was a read or write operation
             switch (_progState) {
                 case IDLE:  // no, just a CV update
-                    if (log.isDebugEnabled()) {
-                        log.debug("Value changed with state IDLE");
-                    }
+                    log.debug("Value changed with state IDLE");
                     return;
                 case READING_FIRST:  // yes, now read second
-                    if (log.isDebugEnabled()) {
-                        log.debug("Value changed with state READING_FIRST");
-                    }
+                    log.debug("Value changed with state READING_FIRST");
                     return;
                 case READING_SECOND:  // now done with complete request
-                    if (log.isDebugEnabled()) {
-                        log.debug("Value changed with state READING_SECOND");
-                    }
+                    log.debug("Value changed with state READING_SECOND");
                     return;
                 default:  // unexpected!
                     log.error("Unexpected state found: {}", _progState);
                     _progState = IDLE;
-                    return;
```

Comment:
Maybe this `return` should be restored as well?

Reply:
I see a pattern, done